python/common/org/python/types/Str.java

```diff
@@ -543,10 +543,66 @@ else if (other instanceof org.python.types.Bool) {
     }
 
     @org.python.Method(
-        __doc__ = ""
-    )
-    public org.python.Object center() {
-        throw new org.python.exceptions.NotImplementedError("center() has not been implemented.");
+        __doc__ = "",
+		args = {"width"},
+		default_args = {"charToFill"}
+    )
+    public org.python.Object center(org.python.Object width, org.python.Object charToFill) {
+		if(this instanceof org.python.types.Str)
+		{
+			java.lang.String fillChar = new java.lang.String();
+			if(charToFill instanceof org.python.types.Str)
```

Comment:
Another style guide thing - we put a space after the if - `if (` - and use K&R style braces (i.e., `if (...) {`).